.travis.yml

```diff
@@ -6,7 +6,7 @@
   go:
     - 1.6.2
   install: make deps
-  script: make .gitvalidation && make validate && make test
+  script: make .gitvalidation && make validate && make test && make test-skopeo
```

Comment:
This means a single Travis run and a single success/fail result.

@runcom do you perhaps off-hand know if it is easy to have the local unit tests and `make test skopeo` show as two separate Github commit checks, so that it would be trivial to tell which one failed?  By no means critical, I have added a `=== Testing skopeo build` marker which could be enough.